### PR TITLE
fix(#1618): display closable prop for Modal docs

### PR DESCRIPTION
**Pull request by syedszeeshan**
The closable property now showing up

image

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
The closable feature (close icon) will always be present in Example Modal, regardless of whether the closable checkbox is checked or unchecked.

image

**Comment by syedszeeshan** [comment]
Hi @lizhuomeng71 , you are right... 

I discussed this with @chrisolsen on Friday before creating the PR...at this time this is how it is...this change just makes sure that the closable porperty does show up in the documentation and when selected it gets appended in the code snippet. 

FYI, React (Sandbox) treats onClose event same as closable=true.

**Comment by lizhuomeng71** [comment]
Hi @syedszeeshan, thanks very much for this update, In that case, this ticket is verified and ready for review, thanks

File: src/routes/components/Modal.tsx [hunk 1]
```diff
@@ -70,6 +70,12 @@ export default function TEMPLATE_Page() {
       options: ["", "fast", "slow", "none"],
       value: "",
     },
+    {
```
Comment (chrisolsen) [inline src/routes/components/Modal.tsx:73]: This property should only be visible if the `lang` is set to `angular`

**Pull request closed without merging**